piper/buttondialog.py

```diff
@@ -220,6 +220,7 @@ def _listbox_filter_func(self, row):
         return True
 
     def _get_button_name_and_description(self, button):
+        # Translators: the {} will be replaced with the button index.
```

Comment:
Yes; each button has an index, e.g. the left mouse button has index 0, the right mouse button has index 1, et cetera. If we map button 1 to button 0, the button with index 0 (left mouse button) will generate the click of button 1 (right mouse button).